### PR TITLE
:sparkles: Generate for URL for SWIFT

### LINKED ISSUE #12: Generate URL for SWIFT
/kind feature

The plugin can create a URL for you in `node-images.yaml` if the user leaves it empty. But if you are using SWIFT then the generated URL will not work. It is because we generate URL like `<endpoint>/<bucket>/<image-name>`, but this not applies to SWIFT.

### PR DESCRIPTION
**Pull request by michal-gubricky**
**What this PR does / why we need it**:
This PR adds generation of node-images url for Swift registry

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #12

### CODE REVIEW

File: example/cluster-stacks/openstack/ferrol/node-images/registry.yaml [hunk 1]
```diff
@@ -4,5 +4,6 @@ config:
   bucket: <bucket_name>
   accessKey: <access_key>
   secretKey: <secret_key>
+  # projectID: <openstack_project_id> # Needs to be specified when type is equal to Swift
```
Comment (chess-knight) [inline example/cluster-stacks/openstack/ferrol/node-images/registry.yaml:7]: If I read the code correctly, `projectID` is necessary only when type equals Swift and URL is not set in `config.yaml`.

File: docs/how_to_use_csctl_plugin_openstack.md [hunk 2]
```diff
@@ -28,7 +28,19 @@ https://<endpoint>/<bucket-name>/<image-dir-name>
 Be aware of that in this method you need to specify `imageDir` in `config.yaml` file.
```
Comment (chess-knight) [inline docs/how_to_use_csctl_plugin_openstack.md:28]: Remove `https://` from the pattern above.
Also, include a pattern for the Swift.

File: example/cluster-stacks/openstack/ferrol/node-images/registry.yaml (hunk 1 above)
Comment (michal-gubricky) [reply example/cluster-stacks/openstack/ferrol/node-images/registry.yaml:7]: Yes, that is right.

File: docs/how_to_use_csctl_plugin_openstack.md [hunk 3]
```diff
@@ -19,16 +19,36 @@ This method can be used when the creator of the cluster-stacks has already built
...
+- for an `S3` type registry:
+
+  ```bash
+  <endpoint>/<bucket-name>/<image-dir-name>
+  ```
+
+- for a `Swift` type registry:
+
+  ```bash
+  <endpoint>/swift/v1/AUTH_/<project-ID>/<bucket-name>/<image-dir-name>
```
Comment (chess-knight) [inline docs/how_to_use_csctl_plugin_openstack.md:33]:
```suggestion
  <endpoint>/swift/v1/AUTH_<project-ID>/<bucket-name>/<image-dir-name>
```

**Review by chess-knight — APPROVED** [review]

**Pull request merged**